benchmark/cluster_containers_setup.py

```diff
@@ -40,33 +38,48 @@ class ContainerOptions(usage.Options):
     optParameters = [
         ['apps-per-node', None, 1, 'Number of application containers per node',
          int],
-        ['app-template', None, None,
-         'Configuration to use for each application container'],
+        ['image', None, None,
+         'Docker image to deploy'],
+        ['mountpoint', None, None,
+         'Location of the mountpoint of the datasets'],
         ['control-node', None, None,
          'Public IP address of the control node'],
         ['cert-directory', None, None,
          'Location of the user and control certificates and user key'],
-        ['wait', None, None,
+        ['max-size', None, 1,
+         'Size of the volume, in gigabytes. One GB by default'],
+        ['wait', None, 72000,
          "The timeout in seconds for waiting until the operation is complete. "
          "No waiting is done by default."],
+        ['wait-interval', None, 4,
+         "How often are we going to check if the creation of containers and "
+         "datasets has finished, in second"],
     ]
 
     synopsis = ('Usage: setup-cluster-containers --app-per-node <containers '
```

Comment:
:question: In theory `Options` is supposed to auto-generate this sort of thing, I thought...